README.md

```diff
@@ -178,6 +178,11 @@ It will also remove the containers and, consequentially, the data held in them.
 buck2 run dev:down
 ```
 
+### (8) Test account credentials
```

Comment:
Suggest to make this step 5 after `Running the Stack` .

Reply:
updated in next commit